content/push/device.textile

```diff
@@ -31,6 +32,21 @@ h3. Configure an Android device
 * Go to *push notifications setup*, click *configure push*.
 * Add your *PEM cert* and *PEM private key*.
 
+h4. Install the Android nuget package for C#.NET
```

Comment:
We don't add the instructions for installing the package for Swift / Obj-C or Android so I think this looks a little out of place?

Reply:
- I have removed these subsections.
- I will re-add along with Swift / Obj-C and Android